src/infrastructure/middleware/authorization/jwt/index.ts

```diff
@@ -46,6 +48,10 @@ class AuthorizationMiddleware {
       applicationStatus.UNAUTHORIZED,
     );
   }
+
+  private authorizationInvalid(next: NextFunction): void {

```

Comment:
Please, check this function that work correctly, why? Because the way that we use the middleware, we are injecting a function, it's the handle, so the context class is going to be lost, and the `this` reference for the function doesn't exists anymore, so an error is going to apear when the function is try to call inside the injected handle function. 
There are to ways to deal with it, use an static function or use and advance feature for JavaScript called bind, and it needs to be used attaching the this after the injection of the handle function. 
```ts
private authorizationInvalid(next: NextFunction): void {
```
or where you are using the middleware 
```ts
this.app.use(myMiddleware.handle.bind(myMiddleware))
```